### PR TITLE
bank app

### PR DESCRIPTION
**Pull request by nutunjamai**
ive got a lot of improving

### CODE REVIEW

File: OnboardingExperience/User.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+
```
Comment (codytwinton) [inline OnboardingExperience/User.cs]: Unnecessary whitespace. Generally only need 1 blank line between your using directives and your namespace.

File: OnboardingExperience/Program.cs [hunk 2]
```diff
@@ -6,9 +6,67 @@ class Program
...
+            Console.WriteLine(questionInt);
+            int num;
+            while (int.TryParse(Console.ReadLine(), out num) == false)
+            {
+                Console.WriteLine("**ERROR** Invalid input enter valid number");
+            }
+            return num;
+        }
+
+        public static bool Accbool(string question)
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: I would change this method name. Something like `AskBoolQuestion`.

File: OnboardingExperience/Program.cs [hunk 3]
```diff
@@ -6,9 +6,67 @@ class Program
...
+                {
+                    var result = AskQuestion(question + " (y/n)");
+
+                    if (result.ToLower() == "y")
+
+                    {
+                        Console.WriteLine("you already have an account");
+                        Console.WriteLine("\n the app will now close");
+
+                        Environment.Exit(-1);//this closes the app
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: I'd move this logic up to your `Main` method.

**Comment by nutunjamai** [comment]
ready for review number 2

**Comment by codytwinton** [comment]
Just commit and push the changes we discussed, and I'll approve this request.

**Comment by nutunjamai** [comment]
changes made ready for approval
